### ISSUE TITLE
Mismatched Titles in Recommendation Sidebar

### ISSUE THREAD

**Issue by DatHypnoboi** [ajayyy/DeArrow#108]
Was rewatching a video when I noticed that the titles of some of the videos in the sidebar were obviously incorrect. 

Title from Game Makers Toolkit on a video from EmpLemon
![Screenshot 2023-07-20 204028](https://github.com/ajayyy/DeArrow/assets/25212257/84b3c959-e29d-4086-bea1-30151bd90db8)

Title from The Film Theorists on a video from elmaxo
![Screenshot 2023-07-20 204041](https://github.com/ajayyy/DeArrow/assets/25212257/439e4d0f-f4a1-49b3-9b07-3c660c162802)

Both the titles were from videos I had watched in the last few days, and I'm subscribed to all 4 channels involved in this, unsure if this is related

**Comment by ajayyy** [comment]
Do you remember the process of getting to that video.

As in, what was the first page loaded (was it directly this video link, or youtube.com), and did you click videos in the sidebar until you got here?

**Comment by ajayyy** [comment]
Also, can you list the other extensions you have installed?

**Comment by DatHypnoboi** [comment]
I got to the video directly from the channel page. I started on subscriptions, clicked on the channel from one of the videos in my sub feed, then went to the channel's video page, then to the video the error occured on. For extensions active on Youtube, I'm using vidIQ, Return Youtube Dislike, SponsorBlock, Thumbnail Rating Bar, and uBlock Origin. I also have DarkReader, but it's disabled on Youtube.
